browser/components/MarkdownPreview.js

```diff
@@ -9,6 +9,7 @@ import SequenceDiagram from 'js-sequence-diagrams'
 import eventEmitter from 'browser/main/lib/eventEmitter'
 import fs from 'fs'
 import htmlTextHelper from 'browser/lib/htmlTextHelper'
+import copy from 'copy-to-clipboard'
```

Comment:
You have to use `clipboard`, built in electron (https://github.com/electron/electron/blob/master/docs/api/clipboard.md), rather than another package.